pkgs/stdenv/generic/setup.sh

```diff
@@ -418,12 +417,12 @@ findInputs() {
 
             # Target offset relative to the package currently being
             # built.
-            mapOffset relTargetOffset
-            local -i targetOffsetNext="$mapOffsetResult"
+            local -i targetOffsetNext
+            mapOffset "$relTargetOffset" targetOffsetNext
 
             # Once again, ensure we're in bounds relative to the
             # package currently being built.
-            [[ "${allPlatOffsets[*]}" = *"$targetOffsetNext"* ]] || continue
+            (( hostOffsetNext <= 1 || hostOffsetNext >= -1 )) || continue
```

Comment:
Should that be an and not or?

Reply:
I did mess it up! :-)

Reply:
I was really split between making it `>1 || <-1` or keeping it like the original `<= 1 && >= -1`
(in the end I just mixed them up)